salt/postgres/backup.sls

```diff
@@ -0,0 +1,16 @@
+include:
+  - postgres
+
+# Require official PostgreSQL repo as they provide a newer version of pgbackrest.
+pgbackrest:
+  pkg.installed: []
```

Comment:
Why an empty array?

Reply:
Salt handles the empty array the same as:
```
pgbackrest:
  pkg.installed
```
Because of the additional `require` under this you need to set `pkg.installed` to something otherwise it would be invalid yaml.

If it isn't clear, we could use this instead
```
pgbackrest:
  pkg.installed: pgbackrest
  require: ... 
```

Reply:
Aha. See `mail.sls` for the preferred style.